notebooks/community/vision/image_warehouse_sdk.ipynb

```diff
@@ -0,0 +1,1062 @@
+{
```

Comment:
<p>All the headings should be in sentence case. Except for the first word, all should be lowercase unless proper noun</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>Done, changed to ## Create and deploy Index</p>